llama-index-core/llama_index/core/callbacks/token_counting.py

```diff
@@ -23,21 +23,61 @@ def __post_init__(self) -> None:
         self.total_token_count = self.prompt_token_count + self.completion_token_count
 
 
+def get_tokens_from_raw_response(raw_response: Dict[str, Any]) -> Tuple[int, int]:
+    """Get the token counts from a raw response."""
+    usage = raw_response.get("usage", None)
+    if usage is None:
+        return 0, 0
+
+    if not isinstance(usage, dict):
+        usage = usage.model_dump()
+
+    possible_input_keys = ("prompt_tokens", "input_tokens")
+    possible_output_keys = ("completion_tokens", "output_tokens")
```

Comment:
Looks like Anthropic uses `input_tokens` and `output_tokens`.

Reply:
Yup!